app/controllers/api/notification_controller.rb

```diff
@@ -0,0 +1,13 @@
+module Api
+  class NotificationController < ApplicationController
+    def show
+      n = current_user.notifications.find(params[:id])
+
+      if n.present?
+        render json: n, status: 200
+      else
+        render nothing: true, status: 400
```

Comment:
don’t think we’ll ever get here, find with raise active record not found and automatically return a 404